.github/workflows/cert_test_checks.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2024 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Certification test checks
+
+on:
+  pull_request:
+    paths:
+      - "src/app/tests/suites/certification/**"
+
+jobs:
+  check-certification-tests:
+    name: Check for common problems in certification tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Checkout submodules & Bootstrap
```

Comment:
bootstrap and submodules seems painful. Would running on the `chip-build` image (which includes python) or using https://github.com/actions/setup-python be faster?

Reply:
done. yes.